source/MRViewer/MRRibbonNotification.cpp

```diff
@@ -25,28 +25,227 @@ constexpr std::array< std::pair<const char*, ImU32>, int( MR::NotificationType::
     std::pair<const char*, ImU32> { "\xef\x8b\xb2", ::MR::Color( 255, 146, 0 ).getUInt32() }
 };
 
+const float cWindowRounding = 4.f;
+const float cWIndowSpacing = 20.f;
```

Comment:
we already have `MRRibbonConstants.h` mb use constants form there

Reply:
fixed